### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authentication_Screen Reader: Focus is going to some invisible content in the page between actions button and deny override button and narrator is reading it as group

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#95]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #ScreenReader; #Narrator; #Chrome; #NVDA; #MAS1.3.1; #MAS1.3.2

**Environment Details:**
URL: https://manage.iis.net/webserver/authentication
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Screen Reader: Narrator, NVDA
MAS ID Violated: MAS 1.3.1, MAS 1.3.2

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Authentication and press enter
6. Verify focus is going to some invisible content in the page between actions button and deny override button and narrator is reading it as group

**Actual Result:**
Focus is going to some invisible content in the page between actions button and deny override button and narrator is reading it as group

**Expected Result:**
Focus should not go to some invisible content in the page between actions button and deny override button

**Note:**
NVDA is reading the invisible content as "clickable"
This issue has been observed in all the pages in the application which has tabs

**User Impact:**
Users with disabilities finds its difficult in working through the application if focus is going to some invisible content in the page and narrator is reading it as group

**Attachments:**
[Bug95_web server_authentication_focus is going to some invisible content.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323454/Bug95_web.server_authentication_focus.is.going.to.some.invisible.content.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
